.renovaterc.json

```diff
@@ -8,5 +8,17 @@
   "dependencies": {
     "enabled": false
   },
-  "ignorePaths": []
+  "ignorePaths": [],
+  "packageRules": [
+    {
+      "updateTypes": ["patch"],
+      "groupName": "dependencies (patch)",
```

Comment:
Nit: Does the group name appear in the PR title? (If so, would Title Case look better?)

Reply:
If only one package under the group is getting upgraded, it remains unchanged:
`📦 Update dependency rollup to v2.10.0`

If multiple dependencies are updated, it uses the group name:
`📦 Update dependencies (patch)`

imo, this looks more natural than title case (and I think we rarely use title case for PRs/commit messages in general. If you feel strongly about this I can adjust, but since you labelled this a "nit" and we'll have a deeper discussion on nomenclature once I create the category grouping configurations, I'm going to leave this as-is for now

Reply:
Aha! I like how you've done it, and retract my suggestion.